v2/dtos/deviceservice.go

```diff
@@ -0,0 +1,24 @@
+//
+// Copyright (C) 2020 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package dtos
+
+// DeviceService represents a service that is responsible for proxying connectivity between a set of devices and the
+// EdgeX Foundry core services.
+// This object and its properties correspond to the DeviceService object in the APIv2 specification:
+// https://app.swaggerhub.com/apis-docs/EdgeXFoundry1/core-metadata/2.x#/DeviceService
+type DeviceService struct {
+	Id             string      `json:"id,omitempty"`
+	Name           string      `json:"name,omitempty"`
+	Created        int64       `json:"created,omitempty"`
+	Modified       int64       `json:"modified,omitempty"`
+	Description    string      `json:"description,omitempty"`
+	LastConnected  int64       `json:"lastConnected,omitempty"`
+	LastReported   int64       `json:"lastReported,omitempty"`
+	OperatingState string      `json:"operatingState,omitempty"`
+	Labels         []string    `json:"labels,omitempty"`
+	Addressable    Addressable `json:"addressable,omitempty"`
```

Comment:
Replace `Addressable` with something else. URL or Address??

Reply:
Host and Port? We currently store (effectively) the URL for the callback endpoint in the Addressable, but although that is the most relevant endpoint for core-metadata, others are available (eg, core-command needs to call the device endpoint)

Reply:
In the Core WG meeting we agreed on BaseAddress which is a fully qualified URI, e.g. `<protocol>:\\<hostname>:<port>/<optional path>`

Reply:
Modified. Replace addressable with baseAddress.